assets/_custom.scss

```diff
@@ -31,6 +31,10 @@ table.data-table {
     &.icon-information {
```

Comment:
Shouldn't `scripts.watch` in `package.json` now be `yarn prebuild && hugo server`? (I got `Failed to get CSV resource "content/en/research/archived-urls.csv"` when running `yarn watch`.)

This is correct in `scripts.build`.